src/index.js

```diff
@@ -1,7 +1,6 @@
 /* eslint-disable */
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
 import { Provider } from 'react-redux';
 import Books from './components/BookStore';
 import store from './redux/configureStore';
```

Comment:
![error](https://user-images.githubusercontent.com/93039359/205460746-ff9436e9-993a-49a7-baca-fc0fe775feb9.PNG)

- [ ] Your application shows an error on startup which is shown above ⬆️ This error is coming because your components folder starts with a capital `C` but you are linking a folder of components with a small `c` Kindly fix this by changing either the folder name or by fixing it here 👍